etl_framework/config_mixins/LoaderMixin.py

```diff
@@ -9,6 +9,7 @@ class LoaderMixin(object):
     LOADER_TABLE_ATTR = 'loader_table'
     LOADER_DELETE_FIELD_ATTR = 'loader_delete_field'
     LOADER_SYNC_FIELD_ATTR = 'loader_sync_field'
+    UPSERT_CONSTRAINT_FIELDS = 'upsert_constraint_fields'
```

Comment:
@ianlofs I'd keep this in UpsertMixin since it's only used for the Upsert statement